src/System.Security.Cryptography.Csp/src/Resources/Strings.resx

```diff
@@ -190,4 +166,4 @@
   <data name="OpenCSP_Failed" xml:space="preserve">
     <value>OpenCSP failed with error code {0}.</value>
```

Comment:
It is used in L136 and L200. But yeah I think that error message isn't helpful at all...